src/sentry/runner/commands/backup.py

```diff
@@ -14,7 +14,13 @@
 @click.argument("src", type=click.File("rb"))
 @configuration
 def import_(src):
-    "Imports data from a Sentry export."
+    """CLI command wrapping the `importer` functionality."""
+
+    importer(src)
```

Comment:
The idea here is that can expose `importer` through the API down the line?

Reply:
Yes, but for now it's just useful for testing. The test suite shouldn't need to call the CLI (mediated through click) to run tests that are fully in-memory.

Reply:
nit: `importer` feels like a weird name for a function since it's a noun and not a verb. Maybe `import_`/`export` for the heavy lifters and `run_{import,export}_command` for the wrappers? Could also move heavy lifters out into their own submodule and keep the function names the same. That's a bigger refactor though so probably don't want to bring that on board this PR.

Reply:
I initially tried this, but I started getting weird errors, leading me to assume that `click` somehow depends on the decorated function matching the name of the command that will be exposed in the CLI? So I went with this naming insteead.

Reply:
Gotcha. In that case maybe `import_` and `_import` and then `export` and `_export`? Leading underscore is a soft "private/internal method" indicator in Python.

Reply:
Went with "exec_import" and "exec_export" for clarity.